hello_world.js

```diff
@@ -1 +1,7 @@
-console.log('hello world');
+
+ document.getElementById("row1").style.background = "blue"
```

Comment:
This works, but lets make this more concise change this to:

```js
var rows = document.getElementsByTagName('tr');

var colorArray = ['', 'blue', 'green', 'red', 'yellow', 'orange']
for (var rowKey = 1; rowKey < rows.length; rowKey++) {
    rows[rowKey].style.backgroundColor = colorArray[rowKey];
}
```
stare at the code a bit until you feel you know whats going on